package.json

```diff
@@ -340,7 +340,7 @@
     "url-search-params-polyfill": "^8.1.0",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#54420e9484592f6838bdf9a278bdd84502e8ef8a",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#bah-contact-info",
```

Comment:
This should reference a commit hash. try using `yarn update:schema`

Reply:
Temporary reference to non-master branch is removed and replaced with correct vets-json-schema hash